client/tracking/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
```

Comment:
Same as what we're doing in WCPay: https://github.com/Automattic/woocommerce-payments/blob/538b322f4576b7cb070332dc627e8a21cfab5c7a/client/tracks/index.js